synapse/rest/media/v1/create_resource.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014-2016 OpenMarket Ltd
+# Copyright 2020-2021 The Matrix.org Foundation C.I.C.
```

Comment:
This is a new file so should be copy righted to `2022` to whatever legal entity you're contributing as